packages/wonder-blocks-modal/components/modal.vite_test.js

```diff
@@ -0,0 +1,101 @@
+/* global render, driver */
+// @flow
+import * as React from "react";
+
+// TODO(kevinb): fix vite so that it can handle importing modules
+// from within the same mono-repo.
+// eslint-disable-next-line monorepo/no-relative-import
+import Button from "../../wonder-blocks-button/index.js";
+
+import ModalLauncher from "./modal-launcher.js";
+import OneColumnModal from "./one-column-modal.js";
+
+jest.mock("../util/needs-hacky-mobile-safari-scroll-disabler.js", () => true);
+
+const TAB = "\ue004";
+
+describe("foo", () => {
+    test("initial focus is the toolbar button", async () => {
+        // Arrange
+        // $FlowFixMe: flow can't resolve render
+        await render(
+            <ModalLauncher
+                modal={
+                    <OneColumnModal
+                        content="Hello, world!"
+                        footer={<Button onClick={() => {}}>Click me!</Button>}
+                    />
+                }
+                opened={true}
+                onClose={() => {}}
+            />,
+        );
+
+        // Act
+        // $FlowFixMe: flow can't resolve driver
+        const element = await driver.switchTo().activeElement();
```

Comment:
To help out flow, is there a way `render` and `driver` can be imported explicitly in the test file from `vite` like we do `enzyme` methods?

Reply:
And in that vein, is there an issue number for fixing this that we can tag these `$FlowFixMe`'s with?

Reply:
It's hard to import these things explicitly b/c there's some setup involved in creating these objects and functions that are being injected into the environment.  Since jest is multi-threaded it runs the environment code when each thread starts up.  I'm working on this in https://github.com/Khan/vite/pull/21.  I think we should probably just fix this the issue with globals in this PR.

Reply:
Yeah, shame there isn't some way of having typed placeholders that can be imported but that themselves call the generated functions. I don't now the vite stuff to know if that makes sense or not.